### DISCUSSION TITLE
It would be nice of more Linux users knew about this tool.  Is there any way to promote it?

### DISCUSSION THREAD

**Discussion by LinuxBeaver** [danielgatis/rembg#79] [Ideas]
This tool has been a massive benefit to my workflow, I use it along side the Gimp.  If there was a Gimp extension or flatpak front end GUI to access this software. There is a good chance more people would learn about it and use it. 

I myself made a bash script to run REMBG from my start menu in a specific folder, but most people are not as tech savy and would complain about a CLI  or bash script using being too hard. I have two ideas that would make REMBG eaiser on the front end.

1. A Gimp Extension. (this is the best idea)

2. A front end GUI to rembg that can be hosted on Flathub.org

The tool would give all REMBG's options and the ability to save images, copy to clipboard or export to Gimp, Krita or whatever.

In the Linux world privacy is huge and this TMK is the first high quality open source neural network that works 100% offline.

I hope this tool can get more attention in the Linux Community, especially the "libregraphics" crowd.

**Comment by colin-adams** [comment]
I've just learned about it. Unforyunately I can't use it as Fedora currently has python 3.11.1
